HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Demo/Scripts/DemoScript.cs

```diff
@@ -39,31 +39,32 @@ IEnumerator getAllPateint()
 
     public void GetPatientByID()
     {
-        StartCoroutine(getPateintByID());
+        StartCoroutine(getPatientByID());
     }
 
-    IEnumerator getPateintByID()
+    IEnumerator getPatientByID()
     {
         Patient patient = new Patient();
-        yield return StorageConnectionServer.GetPatient(patient, "666da72f-1dfa-427a-96a9-c9fb30bf7296");
+        yield return HoloStorageClient.GetPatient(patient, "666da72f-1dfa-427a-96a9-c9fb30bf7296");
         Single.SetActive(true);
         try
         {
-            SinglePatientInfo.text = string.Format("Patient name: \n{0}\nGender: {1}\nDate of Birth: \n{2}", patient.name.full, patient.gender, patient.birthDate.Substring(0, 10));
+            SinglePatientInfo.text = $"Patient name: \n{patient.name.full}\nGender: {patient.gender}\nDate of Birth: \n{ patient.birthDate.Substring(0, 10)}";
         }
         catch (Exception e)
         {
-            Debug.Log("Failed to set the text! \n[Error message]" + e.Message);
+            Debug.LogError("Failed to set the text! \n[Error message]" + e.Message);
         }    
     }
 
     public void LoadModel()
     {
-        ModelSetting.SetManipulable(true);
-        ModelSetting.SetRotation(new Vector3(0, 180, 0));
-        ModelSetting.SetPostition(new Vector3(0.22f, -0.2f, 0.8f));
-        ModelSetting.SetSize(0.12f);
-        StorageConnectionServer.LoadHologram("hololensid");
+        HologramInstantiationSettings setting = new HologramInstantiationSettings();
+        setting.Manipulable = true;
+        setting.Rotation = new Vector3(0, 180, 0);
+        setting.Position = new Vector3(0.22f, -0.2f, 0.8f);
+        setting.Size = 0.12f;
+        HoloStorageClient.LoadHologram(setting, "hid");
```

Comment:
Minor issue, but could you make the setting the second argument, and also [make it optional ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#optional-arguments)with some reasonable default value?